pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -201,4 +202,6 @@ message TLSClientCertificateAuthenticationPolicy {
   // PEM data for the certificate authorities that should be used to
   // validate the remote TLS client.
   string client_certificate_authorities = 1;
+  // Spiffe
+  buildbarn.configuration.spiffe.SubjectMatcher spiffe = 10;
```

Comment:
Buildbarn's story around authentication & authorization is far from perfect (mainly due to historical reasons). That said, do we think it actually makes sense to do subject matching as part of the authentication code? Couldn't we argue it's purely an authorization problem?